src/components/navigation/navigationItem.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+  import type { SvelteHTMLElements } from 'svelte/elements'
+  import type { IconName } from '../../../icons/meta'
+  import Icon from '../icon/icon.svelte'
+
+  type Href = $$Generic<string | undefined>
```

Comment:
I suspect this doesn't need to be `Generic` - it can just be `string | undefined`.

Reply:
In fact, I don't think we need the `href` for anything else, so you can just use the default props from the `a` element, and remove all references to the `href` in this class :smile:

Reply:
What if the item is a button, though?